sdk/src/main/scala/com/horizen/account/api/rpc/service/EthService.scala

```diff
@@ -242,7 +246,7 @@ class EthService(val scNodeViewHolderRef: ActorRef, val nvtimeout: FiniteDuratio
       // other exceptions are not caught as the call would not succeed with any amount of gas
       val check = (gas: BigInteger) => try {
         params.gas = gas
-        doCall(nodeView, params, tag) { (_, _) => true }
+        doCall(nodeView, params, tag, true) { (_, _) => true }
```

Comment:
I think we actually want the exception mapping to happen here too. We just want it outside the binary search, not inside where it would mask the `OutOfGas` exception.

Reply:
See the other comment about moving the exception mapping.